app/models/manageiq/providers/ovirt4/infra_manager.rb

```diff
@@ -0,0 +1,29 @@
+require 'ovirtsdk4'
+
+class ManageIQ::Providers::Ovirt4::InfraManager < ManageIQ::Providers::InfraManager
```

Comment:
The reason for introducing a new base class is that we want to move from the current `Redhat` provider to a new `Ovirt4` provider, step by step, and eventually remove completely the existing `Redhat` classes just removing the files.